### DISCUSSION TITLE
My education pack is approved but still i am not able to redeem benefits

### DISCUSSION THREAD

**Discussion by fletcherfernandes** [community/community#118263] [GitHub Education]
### Hi everyone,

My education pack is approved but still i am not able to redeem benefits.

**Comment by samus-aran** [comment]
Hey @fletcherfernandes  👋

When you submit your application, you'll have your academic affiliation verified and gain access to Global Campus right away. Your educational benefits will arrive within 72 hours of this verification. However, during busy periods like the start of a new semester, it may take a bit longer.

You’ll get an email notification once your academic benefits are ready, or you can check your status [here](https://education.github.com/discount_requests/pack_application). Meanwhile, as a verified [GitHub Global Campus](https://education.github.com/globalcampus/student) student, you can explore the platform, connect with a local [Campus Expert](https://education.github.com/students/experts), attend events hosted by student leaders and GitHub Education, discover and contribute to student-created repositories from [GitHub Community Exchange](https://docs.github.com/en/education/explore-the-benefits-of-teaching-and-learning-with-github-education/github-global-campus-for-students/about-github-community-exchange) and much more!
